CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Added a new validation (`PA137`) to **validate**, ensuring the absence of non-ig
 * Updated the **format** command to use the content graph instead of the id_set file.
 * Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
 * Removed a redundant debug log on the `tools.get_file` function.
+* Added the *auto-replace-uuids* flag to the **download** command. Use it to ... <what Yuval suggested>
```

Comment:
`<...>` 🥲

Reply:
After applying the [suggested change](https://github.com/demisto/demisto-sdk/pull/3615#discussion_r1321108070).
I was fixed a commit after